metrics/density/docker_memory_usage.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test launches a number of containers in idle mode,
+#  It will then sleep for a configurable period of time to allow
+#  any memory optimisations to 'settle, and then checks the
+#  amount of memory used by all the containers to come up with
+#  an average (using the PSS measurements)
+#  This test uses smem tool to get the memory used.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Busybox image: Choose a small workload image, this is
+# in order to measure the runtime footprint, not the workload
+# footprint.
+IMAGE='busybox'
```

Comment:
If we had a stable base image we used for all tests and metrics, then I think that would make sense - but, we have a number of different images we use for different purposes (like alpine here: https://github.com/clearcontainers/tests/blob/master/docker.go#L30 as well as busybox and some network specific ones here: https://github.com/clearcontainers/tests/blob/master/metrics/network/network-metrics-iperf3.sh#L28 and IO one here: https://github.com/clearcontainers/tests/blob/master/metrics/storage/fio_job.sh#L42)

So, I don't think it's quite as simple as defining one image name in the versions.txt and be done with it. Thus, I'm going to propose we leave alone, if you concur?